ui/src/components/SignOutConfirmPage.tsx

```diff
@@ -0,0 +1,43 @@
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import styled from "@mui/material/styles/styled";
+import React, { FC } from "react";
+import { SignOutConfirmPageData } from "src/types";
+
+type SignOutConfirmPageProps = {
+  data: SignOutConfirmPageData;
+};
+const SignOutConfirmPage: FC<SignOutConfirmPageProps> = ({ data }) => {
+  function handleClickCancel(evt: React.MouseEvent) {
+    evt.preventDefault();
+    history.back();
```

Comment:
where does history come from? we dont have a router as far as i know.

Reply:
history is a standard DOM object: https://developer.mozilla.org/en-US/docs/Web/API/History